catalog/admin/admin-poller-service/src/main/java/org/codice/ddf/catalog/admin/poller/AdminPollerServiceBean.java

```diff
@@ -107,7 +107,7 @@ public AdminPollerServiceBean(ConfigurationAdmin configurationAdmin) {
         try {
             objName = new ObjectName(AdminPollerServiceBean.class.getName() + SERVICE_NAME);
         } catch (MalformedObjectNameException e) {
-            LOGGER.error("Unable to create Admin Source Poller Service MBean with name [{}].",
+            LOGGER.info("Unable to create Admin Source Poller Service MBean with name [{}].",
```

Comment:
I don't believe that we would have noticed the error message anyways with the flood of messages that we had previously. I'll write a ticket to do mbean tests.